src/components/JournalPage/index.jsx

```diff
@@ -29,7 +31,7 @@ class JournalPage extends React.Component {
                   return <div>RichText</div>;
                 case RAW_HTML:
                   // return <RawHTML />
-                  return <div>RawHTML</div>;
+                  return <RawHTML content={el.value} />;
```

Comment:
maybe change the case statement so RICH_TEXT and RAW_HTML both go to the RawHTML component as they should both render the same. Can do that in another PR if preferred